payments-core/src/main/java/com/stripe/android/networking/StripeApiRepository.kt

```diff
@@ -78,6 +79,7 @@ internal class StripeApiRepository @JvmOverloads internal constructor(
     private val appInfo: AppInfo? = null,
     private val logger: Logger = Logger.noop(),
     private val workContext: CoroutineContext = Dispatchers.IO,
+    private val productUsageTokens: Set<String> = emptySet(),
```

Comment:
Is it ok from productUsageTokens to not be set?

Reply:
Yeah, for the legacy integrations the `productUsageTokens` will come from `PaymentMethodCreateParams`/`SourceParams`/ `TokenParams`. I'm not changing those because it would be a much bigger change, so I'm keeping support for them by adding all tokens as you can see in `maybeAddPaymentUserAgent`.